public/app/features/dashboard/components/DashboardSettings/TimePickerSettings.tsx

```diff
@@ -80,10 +80,7 @@ export class TimePickerSettings extends PureComponent<Props, State> {
           refreshIntervals={this.props.refreshIntervals}
           onRefreshIntervalChange={this.props.onRefreshIntervalChange}
         />
-        <Field
-          label="Now delay now"
-          description="Enter 1m to ignore the last minute. It might contain incomplete metrics."
-        >
+        <Field label="Now delay" description="Exclude recent data that may be incomplete">
```

Comment:
```suggestion
        <Field label="Now delay" description="Exclude recent data that might be incomplete">
```
Use might to express possibility. See https://developers.google.com/style/word-list?hl=en#might.